package.json

```diff
@@ -70,7 +70,9 @@
     "reformat": "npm run reformat --workspaces --if-present",
     "prepare": "husky",
     "precommit": "precommit",
-    "preinstall": "node scripts/sort-workspaces.js"
+    "preinstall": "node scripts/sort-workspaces.js",
+    "bump-packages": "npm run bump-packages --workspace @mongosh/build",
+    "publish-packages": "lerna publish from-package --no-verify-access --no-push --no-git-tag-version --yes"
```

Comment:
This is 1:1 same as compass, I assume should be fine here also with our lerna setup, unless there's some special option we should pass for mongosh packages.